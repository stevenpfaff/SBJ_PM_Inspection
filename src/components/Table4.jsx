import { Table } from 'react-bootstrap'

import tasks from '../data/table4.json'

const Table4 = () => (
  <Table striped bordered hover>
    <thead>
      <tr>
        <th>Task</th>
        <th>OK</th>
        <th>Location</th>
      </tr>
    </thead>
    <tbody>
      {tasks.map((task, idx) => (
        <tr key={idx}>
          <td>{task.name}</td>
          <td>
            <input type='checkbox' />
          </td>
          <td></td>
        </tr>
      ))}
    </tbody>
  </Table>
)

export default Table4
