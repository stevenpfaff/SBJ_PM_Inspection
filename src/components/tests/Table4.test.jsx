import { render } from '@testing-library/react'

import Table4 from '../Table4'

describe('Table4', () => {
  it('should match snapshot', () => {
    const { container } = render(<Table4 />)

    expect(container).toMatchSnapshot()
  })
})
