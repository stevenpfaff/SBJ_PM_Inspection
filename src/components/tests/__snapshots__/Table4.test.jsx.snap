// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Table4 should match snapshot 1`] = `
<div>
  <table
    class="table table-striped table-bordered table-hover"
  >
    <thead>
      <tr>
        <th>
          Task
        </th>
        <th>
          OK
        </th>
        <th>
          Location
        </th>
      </tr>
    </thead>
    <tbody>
      <tr>
        <td>
          Engine Leaks
        </td>
        <td>
          <input
            type="checkbox"
          />
        </td>
        <td />
      </tr>
      <tr>
        <td>
          Air Leaks
        </td>
        <td>
          <input
            type="checkbox"
          />
        </td>
        <td />
      </tr>
      <tr>
        <td>
          Transmission Leaks
        </td>
        <td>
          <input
            type="checkbox"
          />
        </td>
        <td />
      </tr>
      <tr>
        <td>
          Hydraulic Leaks
        </td>
        <td>
          <input
            type="checkbox"
          />
        </td>
        <td />
      </tr>
      <tr>
        <td>
          Axle Leaks
        </td>
        <td>
          <input
            type="checkbox"
          />
        </td>
        <td />
      </tr>
    </tbody>
  </table>
</div>
`;
